pkg/kotsadm/objects/minio_objects.go

```diff
@@ -35,6 +46,33 @@ func MinioStatefulset(deployOptions types.DeployOptions, size resource.Quantity)
 		securityContext = psc
 	}
 
+	cpuRequest, cpuLimit := "50m", "100m"
+	memoryRequest, memoryLimit := "100Mi", "512Mi"
```

Comment:
Would we need to increase the memory limit for the initcontainers since we're running minio and mc in the same container?

Reply:
likely fine and an OOM situation would be recoverable since it would occur before any data is moved in/out of the PVC, so the memory limit could be patched and the migration would be re-run